.github/workflows/deploy_production.yaml

```diff
@@ -34,7 +36,7 @@ jobs:
         run: |
           mkdir -p ~/.aws
           echo -e "[default]\nregion=eu-central-1" > ~/.aws/config
-          echo -e "[default]\naws_access_key_id=${{ secrets.PRODZAPPA_AWS_ACCESS_KEY_ID }}\naws_secret_access_key=${{ secrets.PRODZAPPA_AWS_SECRET_ACCESS_KEY }}" > ~/.aws/credentials
```

Comment:
After this is merged, we can remove the [secrets](https://github.com/Open-EO/openeo-sentinelhub-python-driver/settings/secrets/actions) that were used for deployment with Zappa

Reply:
Agreed, we can also remove it once we delete the lambda functions.